packages/peregrine/src/Router/Router.js

```diff
@@ -1,33 +1,49 @@
-import React, { Component, createContext } from 'react';
+import React, { Component } from 'react';
 import { BrowserRouter, Route } from 'react-router-dom';
 import { func, object, string } from 'prop-types';
-
-export const { Consumer, Provider } = createContext();
+import MagentoRouteHandler from './MagentoRouteHandler';
 
 export default class MagentoRouter extends Component {
     static propTypes = {
         apiBase: string.isRequired,
         routerProps: object,
-        using: func // e.g., BrowserRouter, MemoryRouter
+        using: func, // e.g., BrowserRouter, MemoryRouter
+        renderRoutes: func,
+        renderMagentoRoutingError: func
```

Comment:
The purpose of the routing updates we made was to not expand the API of this router. This is counter to that purpose.